include/powsybl/iidm/ExtensionProviders.hpp

```diff
@@ -39,13 +40,13 @@ class ExtensionProviders {
 private:
     ExtensionProviders() = default;
 
-    ExtensionProviders(const ExtensionProviders& other) = default;
+    ExtensionProviders(const ExtensionProviders& other) = delete;
```

Comment:
It is needed because exporting the symbol force the generation of every methods, not only called one (https://stackoverflow.com/a/51033485). So the copy constructor was generated, but compilation fails because its tried to copy std::unique_ptr<T> stored in the map attribute. 

Ok to delete copy/move constructors and operators of this class.

> clang-tidy: the delete method should be declare public.

Do you moving to `public` section every copy/move constructors and operators ?